Library/UIView+AutoLayout.swift

```diff
@@ -7,6 +7,13 @@ public func ksr_addSubviewToParent() -> ((UIView, UIView) -> (UIView, UIView)) {
   }
 }
 
+public func ksr_insertSubviewInParent(at index: Int) -> ((UIView, UIView) -> (UIView, UIView)) {
```

Comment:
I just realized we don't have tests for these? Shouldn't be so hard to add?

Reply:
We actually don't have tests for any of these functions, but sure I can add one.